src/libs/actions/Report.ts

```diff
@@ -694,6 +694,7 @@ function updateGroupChatAvatar(reportID: string, file?: File | CustomRNImageMani
             key: `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
             value: {
                 avatarUrl: file?.uri ?? '',
+                avatarFileName: file ? file?.name ?? '' : null,
```

Comment:
```suggestion
                avatarFileName: file?.name ?? '',
```

Reply:
> While the avatarUrl will be removed, the avatarFileName will remain an empty string

How is that the url is being removed but the filename is not? given that both fallback to an empty string, or is it because the former is cleared from BE onyx response?

Either way, let's set both to `null`